test/contrib/dynamodb_test.py

```diff
@@ -0,0 +1,28 @@
+
+import boto3
+from botocore.exceptions import ClientError
+
+from helpers import unittest
+from luigi.contrib.dynamodb import DynamoDBTarget
+from moto import mock_dynamodb2
+
+
+@mock_dynamodb2
+class TestDynamoClient(unittest.TestCase):
+
+    client = boto3.resource('dynamodb', region_name='eu-west-1')
```

Comment:
Differs from Travis default... why?

Reply:
No particular reason, the client is mocked anyway. I'll remove the region from the client construction and let it resolve itself from the ENV var (as per .travis.yml)